### ISSUE TITLE
Compatibility with Debian 8.2 "jessie"

### ISSUE THREAD

**Issue by Emasoft** [rickard2/grsecurity-Debian-Installer#23]
I need to install grsecurity on a Debian 8.2 "jessie" machine with Liquid Feedback. Is it supported by the installer?

**Comment by rufoa** [comment]
Yes - but Debian sid now has a prebuilt grsec kernel package so it's easier to use that (add sid to your apt sources and set the target release to jessie)

**Comment by Emasoft** [comment]
Unfortunately SID is unstable and I need a stable release. What do you suggest?

**Comment by rufoa** [comment]
Set the target release to Jessie in your apt config so sid is only used for this one package. It uses the "testing" patch from grsecurity.net and not unstable as such

**Comment by Emasoft** [comment]
Ok, I'll try and report back.
